enginetest/queries/script_queries.go

```diff
@@ -5277,6 +5277,134 @@ CREATE TABLE tab3 (
 			},
 		},
 	},
+	{
+		Name: "group by having with conflicting aliases test",
+		SetUpScript: []string{
+			"CREATE TABLE tab2(col0 INTEGER, col1 INTEGER, col2 INTEGER);",
+			"INSERT INTO tab2 VALUES(15,61,87);",
+			"INSERT INTO tab2 VALUES(91,59,79);",
+			"INSERT INTO tab2 VALUES(92,41,58);",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query: `SELECT - col2 AS col0 FROM tab2 GROUP BY col0, col2 HAVING NOT + + col2 <= - col0;`,
```

Comment:
would any subset of these be useful planbuilder tests that verify refs get the right definition id?

Reply:
added some of these to planbuilder